docs/.vuepress/config.js

```diff
@@ -150,6 +150,7 @@ module.exports = {
           collapsable: false,
           children: [
             "/dev/guide/quickstart.md",
+            "/dev/guide/contract-verification.md",
```

Comment:
Please, put it after the hello world tutorial (the tutorial that explains how to deploy contracts should go before the tutorial that explains how to verify a deployed contract